akka-http-core/src/main/scala/akka/http/scaladsl/Http.scala

```diff
@@ -215,6 +215,10 @@ class HttpExt private[http] (private val config: Config)(implicit val system: Ex
    *
    * To configure additional settings for a server started using this method,
    * use the `akka.http.server` config section or pass in a [[akka.http.scaladsl.settings.ServerSettings]] explicitly.
+   *
+   * You can pass a [[akka.http.scaladsl.server.Route]] into this function if you
```

Comment:
Wouldn't you get an error about no materializer anyway, because this method also needs an implicit Materializer?

Reply:
No, indeed, that error seems to be masked by the fact that you don't even get a handler.